### PR TITLE
[Merged by Bors] - Getting started docs

### LINKED ISSUE #227: Docs: Getting started Guide
part of this epic: https://github.com/stackabletech/documentation/issues/237

Acceptance criteria:
- [ ] a new module `getting_started` exists, with an `index.adoc`, a `installation.adoc` and `first_steps.adoc`
- [ ] The old `installation.adoc` is removed.
- [ ] All the `yaml` and shell snippets are in the `examples/code` directory and can be executed as a script, to test the documentation
- [ ] versions to install are templated using the `template_docs.sh` script and the `templating_vars.yaml` file.
- [ ] Any example files that are incorporated into the getting started guide are removed from the examples directory (because there they are not tested, with the new script they will be tested).

### PR DESCRIPTION
**Pull request by adwk67**
# Description

Added getting started documentation. Closes #227.

## Review Checklist

- [x] Documentation added (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by maltesander — CHANGES_REQUESTED** [review]
Tested and works. Some findings:

- You can delete the ROOT/nav.doc link to the installation.adoc and the ROOT/pages/installation.adoc file.
- The getting_started module is missing in the antora.yaml
- I would refactor all the `Zookeeper` occurances in the text to `ZooKeeper` (be careful with the kind in the yaml files though)

File: docs/modules/getting_started/pages/installation.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+The tool will show:
+
+[source]
+----
+[INFO ] Installing commons operator
+[INFO ] Installing secret operator
+[INFO ] Installing zookeeper operator
+[INFO ] Installing hdfs operator
+[INFO ] Installing hbase operator
+----
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/pages/installation.adoc]: Thanks! - corrected.

File: docs/modules/getting_started/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+This guide will get you started with HBase using the Stackable operator. It will guide you through the installation of the operator and its dependencies, setting up your first HBase cluster and verifying its operation.
+
+== Prerequisites
+
+You will need:
+
+* a Kubernetes cluster
+* kubectl
+* optional: Helm
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/pages/index.adoc]: Good point - `curl` is not needed locally but `jq` is: added accordingly.

File: docs/modules/getting_started/examples/code/getting_started.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script contains all the code snippets from the guide, as well as some assert tests
+# to test if the instructions in the guide work. The user *could* use it, but it is intended
+# for testing only.
+# The script will install the operators, create a superset instance and briefly open a port
+# forward and connect to the superset instance to make sure it is up and running.
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/examples/code/getting_started.sh.j2]: Done.

File: docs/modules/getting_started/examples/code/hdfs.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    hadoop.log.file=hadoop.log
+    # Define the root logger to the system property "hadoop.root.logger".
+    log4j.rootLogger=${hadoop.root.logger}, EventCounter
+    # Logging Threshold
+    log4j.threshold=ALL
+    log4j.appender.console=org.apache.log4j.ConsoleAppender
+    log4j.appender.console.target=System.err
+    log4j.appender.console.layout=org.apache.log4j.PatternLayout
+    log4j.appender.console.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
+    log4j.appender.EventCounter=org.apache.hadoop.log.metrics.EventCounter
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/examples/code/hdfs.yaml]: Done/removed.

**Comment by adwk67** [comment]
> Tested and works. Some findings:
> 
> * You can delete the ROOT/nav.doc link to the installation.adoc and the ROOT/pages/installation.adoc file.
> * The getting_started module is missing in the antora.yaml
> * I would refactor all the `Zookeeper` occurances in the text to `ZooKeeper` (be careful with the kind in the yaml files though)

Done.

File: docs/modules/getting_started/pages/first_steps.adoc [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+[source,json]
+{
+   "table" : [
+      {
+         "name" : "users"
+      }
+   ]
+}
+
+An alternative way to interact with HBase is to use the https://phoenix.apache.org/index.html[Phoenix] library that is pre-installed on the Stackable HBase image. Use the python utility `psql.py` to create, populate and query a table called `WEB_STAT`:
```
[the remark this answers is not in the document]
Comment (maltesander) [reply docs/modules/getting_started/pages/first_steps.adoc:142]: I did not know this, can we add a hint where to find this?

[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/pages/first_steps.adoc]: I've added a note re. location.

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by adwk67** [comment]
bors merge

**Comment by adwk67** [comment]
bors cancel

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**